mautic/mautic
# Gated video - support for iframe tag

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/353
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

In Europe due to GDPR every visitor is able to disable any tracking script. If disable `mtc.js` then YouTube/Vimeo gated videos do not play, because by default gated video use video tag, what does not support external links. This PR add support for Iframe tag



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Add gated video for youtube/video to the page https://docs.mautic.org/en/gated-video without tracking code (`mtc.js`)
3. See like you re not able to play it
4. After PR you are able to play video and If you add tracking code, then it's converted to custom player with gated video support
